x-pack/plugins/apm/public/components/app/Main/routeConfig.tsx

```diff
@@ -6,25 +6,35 @@
 
 import React from 'react';
 import { Redirect } from 'react-router-dom';
-import ServiceOverview from '../ServiceOverview';
+import { legacyDecodeURIComponent } from '../../../utils/url';
 import ErrorGroupDetails from '../ErrorGroupDetails';
 import ErrorGroupOverview from '../ErrorGroupOverview';
 import TransactionDetails from '../TransactionDetails';
 import TransactionOverview from '../TransactionOverview';
-import { legacyDecodeURIComponent } from '../../../utils/url';
+import { Home } from './Home';
+
+interface BreadcrumbArgs {
+  match: {
+    params: StringMap<any>;
+  };
+}
+
+interface RenderArgs {
+  location: StringMap<any>;
+}
```

Comment:
I just saw the third party breadcrumb helper we use does expose some types. Can we use those (haven't checked so maybe the answer is no)
https://github.com/tsumug/react-router-breadcrumbs-hoc/blob/master/types/react-router-breadcrumbs-hoc/index.d.ts